ThingIFSDK/ThingIFSDK/ThingIFAPI+Trigger.swift

```diff
@@ -8,6 +8,26 @@
 
 import Foundation
 
+extension Predicate {
+    func validate() -> ThingIFError? {
```

Comment:
Let's implement validate login in each classes.
(Would like to follow OOP standard)